apps/examples/testcase/ta_tc/media/utc/utc_media_mediarecorder.cpp

```diff
@@ -289,6 +289,69 @@ class RecorderTest : public MediaRecorderObserverInterface
 	std::condition_variable cvError;
 };
 
+static int get_size()
+{
+	int ret;
+	FILE *fp;	
+	fp = fopen(filePath, "r");
+	if (fp < 0) {
+		meddbg("file open failed error : %d\n", errno);
+		ret = -1;
+		return ret;
+	}
+	fseek(fp, 0, SEEK_END);
+	ret = ftell(fp);
+	fclose(fp);
+	return ret;
+}
+
+static void utc_media_MediaRecorder_setDuration_p(void)
+{
+	MediaRecorder mr;
+	int second = 2;
+	unique_ptr<FileOutputDataSource> dataSource = unique_ptr<FileOutputDataSource>(new FileOutputDataSource(channels, sampleRate, pcmFormat, filePath));
+
+	auto observer = std::make_shared<RecorderTest>();
+	mr.create();
+	mr.setObserver(observer);
+	mr.setDataSource(std::move(dataSource));
+
+	TC_ASSERT_EQ("utc_media_MediaRecorder_setDuration", mr.setDuration(second), RECORDER_OK);
+	
+	mr.prepare();
+	mr.start();
+	observer->waitStarted();
+	mr.stop();
+	mr.unprepare();
+	mr.destroy();
+	
+	TC_ASSERT_EQ("utc_media_MediaRecorder_setDuration", get_size(), second * 4096);
```

Comment:
instead of 4096, calculating needed.
Basically 1 frame size is format of pcm(convert to byte needed) * channel
In this case 2 * 2 = bytes but we using 16k so finally 16 * 2 * 2  = 64k.
If file size was 4K, then it is very strange.